app/main.py

```diff
@@ -1,4 +1,8 @@
 def get_human_age(cat_age: int, dog_age: int) -> list:
+    if not isinstance(dog_age, int) or not isinstance(cat_age, int):
+        raise ValueError
```

Comment:
When raising a `ValueError`, it's important to provide a descriptive message to indicate why the error is being raised. Consider adding a message like 'Invalid input type: age must be an integer'.